api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
That's not what I asked. I asked what will happen to routes already in the database, whose format is default because the field format is empty. Do you see the problem?